docs/installation.md

```diff
@@ -40,3 +40,34 @@ The base installation requires the MuJoCo physics engine (with [mujoco](https://
    ```sh
    $ python robosuite/demos/demo_random_action.py
    ```
+
+### Installing on Windows
+
+It is common to run into issues when installing Robosuite on a Windows machine. Robosuite can be installed on Windows using the following steps.
+
+1. Either follow step 1 from the section [Install from pip](#install-from-pip) or steps 1 and 2 in the section [Install from source](#install-from-source). During this process, you may run into some errors. Please refer to the steps below on how to fix these.
+
+2. If you run into the error `FileNotFoundError: [Errno 2] No such file or directory: 'C:\\tmp\\robosuite.log'`, create a directory called `tmp` under `C:\`.
+
+3. You will also likely face the issue of `mujoco.dll not found`. If you are running in a conda environment (highly recommended), go to the location where your packages are installed (i.e. site-packages). Find the mujoco package. If the mujoco package does not already exists, install it by running 
```

Comment:
"go to the location where your packages are installed (i.e. site-packages)"
let's give some more hints / tips on where they can find this location if they're unsure

Reply:
the official spelling is MuJoCo rather than all lowercase. please fix all instances of this